packages/ui/styles/mixins.less

```
@@ -370,6 +369,16 @@
 	});
 }
 
+// `length()` below refers to specifying font-weight ranges: `600 900`.
+// Ex: 
+//   .buildFontFace("fontName"; local("name"); 400 600); 
+//   -> .buildFontFace("fontName"; local("name"); 400); .buildFontFace("fontName"; local("name"); 600);
+.buildFontFace(@name; @src; @weight: normal) when (length(@weight) > 1) {
+	each(@weight, {
+		.buildFontFace(@name; @src; @value);
+	});
+}
+
```

I think it should be placed before "default". And it need "@style" to support
".buildFontFace(@font-family-base-name; @font-web-src-th-locale-normal; 300 400 600 700; italic);"


```suggestion
// `length()` below refers to specifying font-weight ranges: `600 900`.
// Ex: 
//   .buildFontFace("fontName"; local("name"); 400 600); 
//   -> .buildFontFace("fontName"; local("name"); 400); .buildFontFace("fontName"; local("name"); 600);
.buildFontFace(@name; @src; @weight: normal; @style: "") when (length(@weight) > 1) {
	each(@weight, {
		.buildFontFace(@name; @src; @value; @style);
	});
}

```

---

It's fixed.